### PR TITLE
add default with gradle

### PR DESCRIPTION
**Pull request by atomfrede**
I think it would be useful to be able to to `jhipster jdl default-gradle`.

### CODE REVIEW

File: default-gradle.jdl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+application {
+  config {
+    buildTool gradle,
```
Comment (mraible) [inline default-gradle.jdl]: I don't think commas are necessary. I've been removing them lately.

**Review by mraible — APPROVED** [review]

**Comment by deepu105** [comment]
Just FYI, all the samples I added are with Gradle & react :wink:

**Comment by deepu105** [comment]
we should alos add more defaults like `default-react`, `default-vue` and so on

**Comment by atomfrede** [comment]
Yes having these would be good.

**Pull request merged**